src/ui/public/visualize/visualize_legend.html

```diff
@@ -3,10 +3,13 @@
     type="button"
     ng-click="toggleLegend()"
     class="kuiCollapseButton legend-collapse-button"
+    aria-label="Toggle legend"
```

Comment:
VoiceOver doesn't announce this label because of [this aria-hidden logic](https://github.com/timroes/kibana/blob/170932af4030c32152f5ebcf28a1d5ffda03236d/src/ui/public/visualize/visualization.html#L22) introduced in https://github.com/elastic/kibana/pull/13573. I think that the warning is still useful, but perhaps we should just remove `aria-hidden="true"` from the legend?

Reply:
Good point, I will remove it, but therefor I would like to add an `aria-label` introducing the legend as such. That would also partly solve the issue you mentioned below, regarding missing context for the user with just legend values.

Reply:
Done